app/serializers/forecast_serializer.rb

```diff
@@ -0,0 +1,30 @@
+class ForecastSerializer
+  def initialize(current_forecast, hourly_forecast, daily_forecast)
+    @current_forecast = current_forecast
+    @hourly_forecast = hourly_forecast
+    @daily_forecast = daily_forecast
+  end
+
+  def data_hash
```

Comment:
Handrolled for now... Might come back and try FAST JSON on a branch. Feels easier to handroll this response at least due to the three objects being passed in.